### PR TITLE
New Aura Sphere animation

### PR DESCRIPTION
**Pull request by Ridaz**

### CODE REVIEW

File: data/graphics.js [hunk 1]
```diff
@@ -1552,6 +1552,58 @@ var BattleMoveAnims = {
 			}, 'accel', 'explode');
 		}
 	},
+	aurasphere: {
+		anim: function(battle, args) {
+			var attacker = args[0];
+			var defender = args[1];
+
+			battle.backgroundEffect('#cc9900',700,.6);
```
Comment (Zarel) [inline data/graphics.js]: You see all this red? It's because you used `.6` instead of `0.6`

**Comment by ascriptmaster** [comment]
Do you have footage of this new version?

**Comment by Ridaz** [comment]
That's the old one. @Zarel said it looked good without the extra graphics so, according to the code; it looks like this:

**Comment by Zarel** [comment]
Oh, I just meant I wanted to know whether or not you added any extra graphics files to `fx/`. I'm fine with or without the extra effects.

**Pull request merged**